Makefile

```diff
@@ -1,5 +1,11 @@
 .PHONY: all lint test init pylint pep8 test-unit test-unit-via-clusterrunner test-functional freeze
 
+NAME=$(shell grep '^Name' .flute | grep -Eo '[[:alnum:]_-]+$$')
+RPM_VERSION_PREFIX=0.5
+COMMIT_COUNT=$(shell git rev-list --count HEAD)
```

Comment:
To run ClusterRunner (i.e. main.py) don't we need to run `make init` first to install requirements?
Why the builder machine should have capability to run the CR executable? It should just package the CR.